README.md

```diff
@@ -1 +1,59 @@
-# hello-react-rails
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-react-rails
+
+> This App is a saple of combination between rails and react
+
+## Built With
+
+- Ruby on Rails
+- PostgresSQL
+- React
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+  - A web editor like Atom or Vs code
+  - Install ruby and rails
+
+### Setup
+  - Copy this link `https://github.com/camilevahviraki/hello-react-rails`.
+  - Get the directory that you want to clone the repository.
+  - Open the command prompt in this directory.
+  - Write `git clone https://github.com/camilevahviraki/hello-react-rails`.
+  - Go to the repository folder in your command prompt `cd recipe-rails`.
+
+### Install
+  - run `bundle install`
+  - run `npm install`
+
+
+### Usage
+  - run `bin/dev` to start the local server or
+
+## Author
+
+👤 **Camile**
+
+- GitHub: [@camilevahviraki](https://github.com/camilevahviraki)
+- Twitter: [@camilevahviraki](https://twitter.com/CamileVahviraki)
+- LinkedIn: [camilevahviraki](https://www.linkedin.com/in/camile-vahviraki)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] OPTIONAL
It's recommended that you add an MIT license, or any other similar license to your ReadMe.

Reply:
Thank you  @rloterh